src/pages/Favspage/Favspage.js

```diff
@@ -25,20 +27,22 @@ class Favspage extends Component {
         this.setState(state => {
           const moviesArray = state.movies.concat(movie);
           return {
-            movies: moviesArray
+            movies: moviesArray,
+            error: ""
```

Comment:
I would name it `errorMessage` because you're caring only about a string, not the error object (i.e `new Error('')`)